### DISCUSSION TITLE
:grey_question:  Need resources for endoflie.date

### DISCUSSION THREAD

**Discussion by adriens** [jreleaser/jreleaser#1083] [General]
cf following [tweet](https://twitter.com/aalmiray/status/1598448842789507075) : 

![image](https://user-images.githubusercontent.com/5235127/205241693-73c39387-5fcf-42fb-9b4b-7769db668ef6.png)

I need the following elements : 

- A releasePolicy markdown on your repo
- Jreleaser icon on https://simpleicons.org/

**Comment by aalmiray** [comment]
Icon request https://github.com/simple-icons/simple-icons/issues/8092

**Reply by aalmiray** [reply]
Release history is now collected at https://jreleaser.org/guide/latest/release-history.html

**Comment by adriens** [comment]
One more question : 

> _We only list stable releases. Was JReleaser 0.x branch considered stable?_

- https://github.com/endoflife-date/endoflife.date/pull/1962#issuecomment-1335313729

**Reply by aalmiray** [reply]
Not really. We broke compatibility during 0.x when it made sense. We've tried to keep strict BC in 1.x (with a minor hiccup in 1.2.0).

**Comment by adriens** [comment]
Seems like I had all the required datas. I'll let you know as soon as the pr will be merged, very soon I guess
